pkg/api/alert_client.go

```diff
@@ -23,6 +71,97 @@ type AlertSettingsConditionPayload struct {
 	Channels    []string `json:"channels" yaml:"channels"`
 }
 
+func constructQueryString(page int, pageSize int, sortField, sortOrder, templateName, channelName string) (query string) {
+	path := fmt.Sprintf("%s?pageSize=%d", pkg.AlertChannelsPath, pageSize)
```

Comment:
I could also use `http/url` helpers to do this:
```
params := url.Values{}
params.Add("foo1", "bar1")
params.Add("foo2", "bar2")
return "?"+params.Encode()
```

Reply:
ncie ^ but with returning an `fmt.Sprintf` not with `+`